src/wristpy/core/computations.py

```diff
@@ -43,7 +43,54 @@ def moving_mean(array: Measurement, epoch_length: int = 5) -> Measurement:
     if array.measurements.ndim == 1:
         measurements_mean_array = measurements_mean_array.flatten()
 
-    return Measurement(
+    return models.Measurement(
         measurements=measurements_mean_array,
         time=measurement_df_mean["time"],
     )
+
+
+def moving_median(
+    acceleration: models.Measurement, window_size: int
+) -> models.Measurement:
+    """Applies moving median to acceleration data.
+
+    Args:
+        acceleration: the three dimensional accelerometer data. A Measurement object,
+        it will have two attributes. 1) measurements, containing the three dimensional
+        accelerometer data in an np.array and 2) time, a pl.Series containing
+        datetime.datetime objects.
+
+        window_size: The size of the overlapping window within which the median will be
```

Comment:
overlapping with what? Isn't it just the size of the window?

Reply:
yes exactly, I'll make that edit.